sdks/go/README.md

```diff
@@ -84,56 +145,237 @@ sentence: 1
 purse: 6
 ```
 
-To run wordcount on dataflow runner do:
+#### Run Example on the Dataflow Runner
```

Comment:
This should definitely only belong in the Go Quickstart or similar tutorial(s). It's not appropriate for this location. 

This is good content, this is just not the place for it.